src/storage/buffer/buffer_obj.cpp

```diff
@@ -47,28 +47,36 @@ BufferObj::~BufferObj() = default;
 BufferHandle BufferObj::Load() {
     std::unique_lock<std::shared_mutex> w_locker(rw_locker_);
     switch (status_) {
-        case BufferStatus::kLoaded:
+        case BufferStatus::kLoaded: {
+            break;
+        }
         case BufferStatus::kUnloaded: {
+            if (!buffer_mgr_->RemoveFromGCQueue(this)) {
+                UnrecoverableError("Invalid call.");
```

Comment:
Error message: attempt to buffer: {path} status is UNLOADED, but not in GC queue.